content/webapp/services/prismic/types/filters.ts

```diff
@@ -1,4 +1,4 @@
-import { predicate } from '@prismicio/client';
+import * as prismic from '@prismicio/client';
```

Comment:
I think that decision might've been made because of custom types that have similar names and it would differentiate them and ensure the dev knows they're working with a Prismic package. I'd get that too, but there would be solutions to it such as
```
import { RichTextField as PrismicRichTextField } from '@prismicio/client'
```

Anyway, not a hill I'd die on, it just kept jumping up to me!